schema/PharmGKBIDs.mcf

```diff
@@ -0,0 +1,176 @@
+# generated by script based on dictionaries from
+# /scripts/pharmgkb/drug_gene_relations/config.py , then modified by hand
+
+Node: dcid:alfredID
```

Comment:
I'll leave it to @spiekos -- typically it's nice to have descriptions for properties so that people browsing the schema know when to reuse. In the case of Bio, I'm not sure how "obvious" alfred, genAtlas, etc. are. But perhaps at least a hint that these are IDs from various bio databases?

Reply:
These were generated by a script. It would require manual work to go and extract URLs and descriptions for each of these script generated properties. I'm ok not having them to avoid doing this.